ui/js/directives/treeherder/clonejobs.js

```diff
@@ -270,21 +270,12 @@ treeherder.directive('thCloneJobs', [
          * display.
          */
         var addGroupJobsAndCounts = function(jgObj, platformGroup) {
-            var ct, job, jobCountBtn, l;
-            var countAdded = false;
             var jobCountBtnArray = [];
             var jobBtnArray = [];
             var stateCounts = {};
             var lastJobSelected = ThResultSetStore.getSelectedJob($rootScope.repoName);
 
-            var jobList = platformGroup.find(".group-job-list");
-            var countList = platformGroup.find(".group-count-list");
-            jobList.empty();
-            countList.empty();
-
-            for (l = 0; l < jgObj.jobs.length; l++) {
-
-                job = jgObj.jobs[l];
+            _.forEach(jgObj.jobs, function(job) {
```

Comment:
I was wondering which was the fastest array looping, and it looks like someone did a compare here: https://jsperf.com/angular-foreach-vs-native-for-loop/28

So, according to this, using ``jgObj.jobs.forEach(`` would be even faster than this.

Granted, the test case uses a slightly older lodash version.  But EC5 might still be faster since it doesn't have to do any evaluation.  This is one of the most speed intensive parts of our client code, so trimming every bit helps.  :)  Your call, though.